internal/zinc-persist/src/main/scala/sbt/internal/inc/text/Base64.scala

```diff
@@ -30,10 +29,28 @@ private[sbt] object Base64 {
   }
 }
 
+private[sbt] object Java678Encoder {
+  private[this] val DatatypeConverter_class = Class.forName("javax.xml.bind.DatatypeConverter")
+  private val DatatypeConverter_encode =
+    DatatypeConverter_class.getMethod("printBase64Binary", classOf[Array[Byte]])
+  private val DatatypeConverter_decode =
+    DatatypeConverter_class.getMethod("parseBase64Binary", classOf[String])
+}
+
 private[sbt] class Java678Encoder extends Base64 {
```

Comment:
BTW, Is there any reasons `Java678Encoder` exists? zinc still support Java 7? 🤔 

https://github.com/sbt/zinc/issues/451#issuecomment-342408977

>  we can target anything less than Java 8 given we depend on JDK 8 classes

Reply:
We can drop Java 7 workarounds.

Reply:
removed